File: app/src/main/java/by/offvanhooijdonk/plaincalendar/widget/helper/PermissionHelper.java
package by.offvanhooijdonk.plaincalendar.widget.helper;

import android.Manifest;
import android.content.Context;

import pub.devrel.easypermissions.EasyPermissions;

public class PermissionHelper {

    public static boolean hasCalendarPermissions(Context ctx) {
        return EasyPermissions.hasPermissions(ctx, Manifest.permission.READ_CALENDAR);
    }
}

File: app/src/main/java/by/offvanhooijdonk/plaincalendar/widget/helper/PermissionHelper.kt
package by.offvanhooijdonk.plaincalendar.widget.helper

import android.Manifest
import android.content.Context
import pub.devrel.easypermissions.EasyPermissions

fun hasCalendarPermissions(ctx: Context): Boolean { //todo move where used
    return EasyPermissions.hasPermissions(ctx, Manifest.permission.READ_CALENDAR)
}

